DevBytes-starter/app/src/main/java/com/example/android/devbyteviewer/ui/DevByteFragment.kt

```diff
@@ -126,7 +126,7 @@ class DevByteFragment : Fragment() {
 
 
         // Observer for the network error.
-        viewModel.eventNetworkError.observe(this, Observer<Boolean> { isNetworkError ->
+        viewModel.eventNetworkError.observe(viewLifecycleOwner, Observer<Boolean> { isNetworkError ->
```

Comment:
Does this part need a codelab change?

Reply:
This was the api modification due to library update, which was done earlier.

Reply:
This is starter app repo, I don't think we need a codelab change